### PR TITLE
v1.7.0 - for CI testing - will push directly when ready

### PR DESCRIPTION
**Pull request by shefty**
Signed-off-by: Sean Hefty <sean.hefty@intel.com>

### CODE REVIEW

**Comment by shefty** [comment]
AWS failure is 
```
12:06:25 - name:   fi_rdm_cntr_pingpong -I 5 -p "udp"
12:06:25   result: Fail
12:06:25   time:   2
12:06:25   server_cmd: /home/ec2-user/workspace/multi-node/PROVIDER/udp/label/rhel/fabtests/install/bin/fi_rdm_cntr_pingpong -I 5 -p "udp" -s 172.31.27.205
12:06:25   server_stdout: |
12:06:25   client_cmd: /home/ec2-user/workspace/multi-node/PROVIDER/udp/label/rhel/fabtests/install/bin/fi_rdm_cntr_pingpong -I 5 -p "udp" -s 172.31.18.26 172.31.27.205
12:06:25   client_stdout: |
12:06:25     fi_getinfo(): common/shared.c:1436, ret=-61 (No data available)
```

**Comment by jsquyres** [comment]
@shefty Will there be a v1.7.x branch?

**Comment by shefty** [comment]
@jsquyres - Yes - I'll push 1.7 changes into master, then create v1.7.x at that point.  Or at least that is my hope.

**Comment by rajachan** [comment]
bot:aws:retest

File: NEWS.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ This file contains the main features as well as overviews of specific
 bug fixes (and other actions) for each version of Libfabric since
 version 1.0.
 
-v1.7.0, Fri Dec 28, 2018
-========================
+v1.7.0, Mon Jan 7, 2018
```
Comment (j-xiong) [inline NEWS.md]: the year should be '2019'.

Comment (shefty) [reply NEWS.md]: Glad someone caught this.  Thanks!

**Pull request merged**